src/components/Questions/AnswerInput.vue

```diff
@@ -145,6 +152,10 @@ export default {
 			})
 		},
 
+		optionDragMenuId() {
+			return 'q' + this.answer.questionId + 'o' + this.answer.id + '__drag_menu'
```

Comment:
I prefer template string, but no blocker ;)
```suggestion
			return `q${this.answer.questionId}o${this.answer.id}__drag_menu`
```